docs/backup_and_restore.rst

```diff
@@ -75,57 +75,42 @@ the SecureDrop servers.
 
 .. code:: sh
 
-   source .venv/bin/activate
-   cd install_files/ansible-base
-   ansible -m ping all
+   ssh app uptime
 
 If this command fails (usually with an error like "SSH Error: data could not be
 sent to the remote host. Make sure this host can be reached over ssh"), you need
 to debug your connectivity before proceeding further. Make sure:
 
-* Ansible is installed (``which ansible`` should return a path instead of "not found").
-
-  * Ansible should be automatically installed by the Tails auto-configuration
-    for SecureDrop. If it is not, you probably need to re-run
-    ``./securedrop-admin tailsconfig``. See
-    :doc:`configure_admin_workstation_post_install` for detailed instructions).
+* Ansible is installed. If it is not, see
+  :doc:`configure_admin_workstation_post_install` for detailed instructions.
 
 * The *Admin Workstation* is connected to the Internet.
 * Tor started successfully.
 * The ``HidServAuth`` values from ``app-ssh-aths`` and ``mon-ssh-aths`` are in
-  Tails' ``/etc/tor/torrc``.
-
-  * Tor should be automatically configured to connect to the authenticated Tor
-    Hidden Services by the Tails auto-configuration for SecureDrop. If it is
-    not, you probably need to re-run ``./securedrop-admin tailsconfig``. See
-    :doc:`configure_admin_workstation_post_install` for detailed instructions).
+  Tails' ``/etc/tor/torrc``. If they are not, again, see 
+  :doc:`configure_admin_workstation_post_install` for detailed instructions.
 
-Run the backup Ansible role
-'''''''''''''''''''''''''''
+Create the backup
+'''''''''''''''''
 
-Now you can run the production Ansible playbook with special flags tag to
-perform the backup:
+Run:
 
 .. code:: sh
 
-   source .venv/bin/activate
-   cd install_files/ansible-base
-   ansible-playbook -t backup securedrop-prod.yml -e perform_backup=true
+   ./securedrop-admin backup
 
-.. todo:: Test this on a real *Admin Workstation*
+The backup action will display itemized progress as the backup is created.
+Run time will vary depending on the number of submissions saved on
+the *Application Server*.
 
-The backup role will print out the results of its tasks as it completes them.
-You can expect the ``fetch the backup file`` step to take a long time,
-especially if you have a lot of saved submissions.
+When the backup action is complete, the backup will be stored as a tar archive in
+``install_files/ansible-base``. The filename will start with ``sd-backup``, have
+a timestamp f when the backup was initiated, and end with ``.tar.gz``. You can
```

Comment:
I'll wait to implement typo fixes and similar till you've finished review, so I can knock them all out at once.